tpp-app/src/info/Info.js

```diff
@@ -1,26 +1,168 @@
+            stigma surrounding periods through public education and advocacy.
+            </Text>
+            <TouchableOpacity
+                style={styles.visitButton}
+                onPress={() => Linking.openURL('https://www.theperiodpurse.com')}
+            >
+                <Text style={{...styles.productText, margin: 10}}>Visit the website</Text>
+            </TouchableOpacity>
+        </View>  
+    )
+}
+
+const PadsCard = () =>{
+    return(
+        <View style={styles.productCard}>
+            <Image style={styles.productIcon} source={padIcon}/>
+            <Text style={styles.productText}> Pads</Text>
+        </View>
+    )
+}
+
+const TamponsCard = () =>{
+    return(
+        <View style={styles.productCard}>
+            <Image style={styles.productIcon} source={tamponsIcon}/>
+            <Text style={styles.productText}>Tampons</Text>
+        </View>
+    )
+
+}
+
+const PeriodUnderwearsCard = () =>{
+    return(
+        <View style={styles.productCard}>
+            <Image style={styles.productIcon} source={underwearIcon}/>
+            <Text style={styles.productText}>Period {'\n'} Underwears</Text>
+        </View>
+    )
+
+}
+
+const MenstrualCupsCard = () =>{
+    return(
+        <View style={styles.productCard}>
+            <Image style={styles.productIcon} source={cupIcon}/>
+            <Text style={styles.productText}>Menstrual Cups</Text>
+        </View>
+    )
+}
+
+const ClothPadsCard = () =>{
+    return(
+        <View style={styles.productCard}> 
+            <Image style={styles.productIcon} source={clothPadIcon}/>
+            <Text style={styles.productText}>Cloth Pads</Text>
+        </View>
+    )
+}
+
 
 
 export default function Info () {
     return (
-        <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
-            <Text>Info!</Text>
-            <Card>
-                <Card.Title>Title</Card.Title>
-                <Card.Divider/>
-                <Text style={{ marginBottom: 10}}>
-                    Reiciendis rem sequi occaecati. Nesciunt dolores hic saepe neque nisi.
-                    Rerum voluptatem autem et. lias voluptatem fugit sunt id repellat.
-                    At pariatur quisquam assumenda possimus aut. Sunt delectus officiis voluptate rerum.
-                    Eum fugiat accusamus ut. Error nostrum maxime omnis et quos occaecati consequuntur.
-                    Repellat omnis autem labore sit ex sequi quia consequatur.
-                </Text>
-            <Button
-                buttonStyle={{ borderRadius: 0, marginLeft: 0, marginRight: 0, marginBottom: 0}}
-                title='Learn more' />
-            </Card>            
-        </View>
+        <ScrollView>
+            <View style={styles.cardContainer}>
+                <Text style={{...styles.productText, textAlign: 'left',
+        color: "#6D6E71",
+        margin: 15}}>Learn more about period products</Text>
+
+                <View style={styles.containerRow}>
```

Comment:
I think you can put all the `Card` things in a single `View`, and set the `flexWrap` property to `wrap`. Since u already have the flexbox going by row, it should just wrap around and create the same effect.

Reply:
That's what I thought too, but it didn't give the same result for some reason. I'll try it again because maybe I did it wrong.

Reply:
Ah, I needed to use flex-basis! Fixed now